### DISCUSSION TITLE
Error - I can't download the results

### DISCUSSION THREAD

**Discussion by Jo1410** [asreview/asreview#634] [Troubleshooting (report errors in the "Issues" tracker)]
I'm currently trying to download the results of my reviewing process in asreview but after clicking on "download" the following error message occures: 
`{"message":"Initializing from file failed"} `

Do you have any idea what I can do?

**Comment by J535D165** [comment]
Thanks for reporting. In what format do you try to export the results? And what is the message you see in the command line?

**Reply by Jo1410** [reply]
Hi @J535D165, 
thanks for your help! 
I tried to download it in both types, excel and CSV, but it doesn't work at all. 
My command windows displays:
`ERROR:root:Initializing from file failed
INFO:werkzeug:127.0.0.1 - - [15/Jun/2021 10:20:33] "←[35m←[1mGET /api/project/value-of-effort-part-2/export?file_type=excel HTTP/1.1←[0m" 500 -`

**Reply by J535D165** [reply]
I'm sorry. Can you send the project file to asreview@uu.nl? You can export it from the project dashboard.

**Reply by Rensvandeschoot** [reply]
It appears @Jo1410 imported all articles into the literature management program "Citavi" to delete the duplicates. Then she exported the dataset as a .ris file for ASReview.  I was able to download the file with labels as a .csv file, but there is a column titled `unknown_tag` with for me unclear code, like: 

```
defaultdict(<class 'list'>, {'U3': ['Journal Article'], 'TS': ['PubMed'], 'PM': ['29477776'], 'S1': ['14'], 'M4': ['Citavi']})
```

It seems the encoding of this columns causes issues . I am not familiar with such code, not with Citavi. Are you @boer0107 ??

**Reply by boer0107-zz** [reply]
I expect that the issue is in the ris file that has been exported from Citavi. Either because Citavi did not export this well or ASR can’t handle it. @ottomattas might be able to help out on this because he’s the expert on ris standards (or the lack of ris standards).

The column unknow tag contains fields from the ris file that weren’t processed properly. They all start with the U3 field (a user definable field). This is probably causing the issues. But also information from other fields end up in this column. So getting rid of the column might also mean getting rid of relevant information.

This is the full record in the ris file for the example that Rens gave: 

TY  - JOUR
AU  - Inzlicht, Michael
AU  - Shenhav, Amitai
AU  - Olivola, Christopher Y.
T1  - The Effort Paradox: Effort Is Both Costly and Valued
JF  - Trends in cognitive sciences
SP  - 337
EP  - 349
IS  - 4
VL  - 22
PY  - 2018
U3  - Journal Article

Research Support, N.I.H., Extramural

Research Support, Non-U.S. Gov't

Review
KW  - Animals
KW  - Cognitive control
KW  - COGNITIVE dissonance
KW  - decision-making
KW  - EFFORT
KW  - Humans
KW  - learned industriousness
KW  - motivation
KW  - need for cognition
KW  - Psychological Theory
KW  - Value
KW  - Volition
L1  - C:\Users\Johanna\AppData\Local\Swiss Academic Software\Citavi 6\ProjectCache\itscd2q1ntiivsaug3npvvkc8aj7wdn06u53ygk3qm\Citavi Attachments\9ae7c95c-2eab-4d96-acd0-5ceb152bcacb.pdf
L2  - https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6172040
AB  - According to prominent models in cognitive psychology, neuroscience, and economics, effort (be it physical or mental) is costly: when given a choice, humans and non-human animals alike tend to avoid effort. Here, we suggest that the opposite is also true and review extensive evidence that effort can also add value. Not only can the same outcomes be more rewarding if we apply more (not less) effort, sometimes we select options precisely because they require effort. Given the increasing recognition of effort's role in motivation, cognitive control, and value-based decision-making, considering this neglected side of effort will not only improve formal computational models, but also provide clues about how to promote sustained mental effort across time.
LA  - eng
TS  - PubMed
DO  - 10.1016/j.tics.2018.01.007
PM  - 29477776
AD  - Department of Psychology, University of Toronto, 1265 Military Trail, Toronto, Ontario M1C 1A4, Canada; Rotman School of Management, University of Toronto, 105 St. George Street, Ontario M56 3E6, Canada. Electronic address: michael.inzlicht@utoronto.ca.

Department of Cognitive, Linguistic, and Psychological Sciences, Brown Institute for Brain Science, Brown University, Providence, RI, USA.

Tepper School of Business, Carnegie Mellon University, Pittsburgh, PA, USA; Department of Social and Decision Sciences, Carnegie Mellon University, Pittsburgh, PA, USA.
S1  - 14
M4  - Citavi
ER  -

**Reply by Rensvandeschoot** [reply]
@ottomattas will you add Citavi to your new comparison?
